### ISSUE TITLE
Blocking open source repositories is a conflict with the any idea of free software and the GNU license.

### ISSUE THREAD

**Issue by pr1v3t** [stop-war-in-ukraine/stop-russia-it#47]
Freedom to run the program as you wish for any purpose (freedom 0) - for any kind of person or organization to use it on any kind of computing system for any kind of work or purpose.

By rejecting the idea of GNU, the whole world is being deprived of freedom.

**Comment by oleksiizabara** [comment]
опять же повторюсь, если тебе раньше не понятно было! фашисты исключение из любых договоренностей между цивилизованными странами и организациями!

**Comment by digiom-ceo** [comment]
> опять же повторюсь, если тебе раньше не понятно было! фашисты исключение из любых договоренностей между цивилизованными странами и организациями!

[ru.wikipedia.org/wiki/Бандера,_Степан_Андреевич](https://ru.wikipedia.org/wiki/%D0%91%D0%B0%D0%BD%D0%B4%D0%B5%D1%80%D0%B0,_%D0%A1%D1%82%D0%B5%D0%BF%D0%B0%D0%BD_%D0%90%D0%BD%D0%B4%D1%80%D0%B5%D0%B5%D0%B2%D0%B8%D1%87)

[uk.wikipedia.org/wiki/Бандера_Степан_Андрійович](https://uk.wikipedia.org/wiki/%D0%91%D0%B0%D0%BD%D0%B4%D0%B5%D1%80%D0%B0_%D0%A1%D1%82%D0%B5%D0%BF%D0%B0%D0%BD_%D0%90%D0%BD%D0%B4%D1%80%D1%96%D0%B9%D0%BE%D0%B2%D0%B8%D1%87)

**Comment by jsisaev1** [comment]
> опять же повторюсь, если тебе раньше не понятно было! фашисты исключение из любых договоренностей между цивилизованными странами и организациями!

переможный зрадовец, тебе давно ГРАДом не поливали? может быть, буратино тебе устроит солнцепек? или может быть, тебе надо подарить букет гиацинтов с тюльпанами? или васильков на подносе? также могу предложить тебе ураганный смерч с пионами и гвоздики. тетушка Нона может предлоэить тебе куст акации в снегу и тунгуску.

**Comment by digiom-ceo** [comment]
Ukrainian Nazis delete comments. There is no democracy.

**Comment by saintdianabless** [comment]
sadly, repo owner is dominated by emotions
we should not pollute GitHub/OSS/... more, this is a very dangerous abuse

**Comment by greatbody** [comment]
> опять же повторюсь, если тебе раньше не понятно было! фашисты исключение из любых договоренностей между цивилизованными странами и организациями!

Do you mean all Russia people, ordinary people, as Nazi?!!!

---
We are human not animal.

This world is for everyone.

Think more than just this earth, go towards the sea of stars.

STOP NATO, and Russia will end the war.

![image](https://user-images.githubusercontent.com/6984599/156688445-6b15d42b-c609-42e1-9233-a98f28dd8505.png)

**Comment by digiom-ceo** [comment]
> Do you mean all Russia people, ordinary people, as Nazi?!!!

Ты спрашиваешь про нацистов у нациста. Он наводит цензуру на любое мнение и ценит только радикалов и нацизм.

**Comment by hauleth** [comment]
> This world is for everyone.

Apparently [not for Ukrainian kindergarten kids](https://www.newsweek.com/russia-attacks-ukraine-kindergarten-orphanage-sumy-oblast-shelter-casualties-reported-1682656).

> STOP NATO, and Russia will end the war.

Russian Federation government started the war, not NATO, not anyone else. It is your government to blame and do not try to confuse the attacker and the defender.

**Comment by cpressey** [comment]
> Blocking open source repositories is a conflict with the any idea of free software and the GNU license. 

No, it is not. Providing network access to repositories is a _service_. The fact that GitHub, npm, etc do this in a “free as in beer” way, is orthogonal to the “free as in freedom” philosophy of GNU.

The GPL obliges only that if you distribute binaries you must also distribute source.

Nothing is obliging these companies to provide these services to any entity, whether that entity is waging an unjustifiable, incompetently conducted, and fundamentally delusional war on another country or not.

**Comment by Arkemlar** [comment]
Let me imagine vice versa situation: Ukraine invaded to Russia. Just imagine. And let's vote.
Vote ONLY if you are NOT impacted directly by conflict (you are not Ukrainean or Russian, you live not in these countries and far away from conflict area, you don't have relatives or friends in both countries, you never been on these countries) then just ask yourself, would you feel equal amount of hate to invader in these mirror situation?
Is your feelings and mindset different? More involved/raged or more indifferent?
Vote happy smile 😄 if you more indifferent. 
Vote sad smile 😕 if you more raged.
Vot eyes 👀 if nothing changed (really??) 
<details>
<summary>Let me guess results</summary>
I am sure, most of the answers wold be about more indifference.
And I am also 100% sure there won't be so much hate, so much effort to block Ukraine and so much will to ban Ukraine normal people from every service around the world.
Why so? I have an answer but I let you think about it by yourself.
</details>

**Comment by webknjaz** [comment]
Why would it be the same? r*ssia is a fascist nazi state, Ukraine is not. Ruscism needs to stop.

**Comment by JeffRockatansky** [comment]
@webknjaz How exactly does violation of software freedom will stop Ruscism?

**Comment by webknjaz** [comment]
It's a social issue that has nothing to do with software freedom. Why are you in favor of giving terrorists a platform for planning genocides, again?

**Comment by NoPlagiarism** [comment]
@Arkemlar "IMAGINE" сука, just "IMAGINE". Sorry, but Invasion is Invasion. We need to think about now and destroy fucking regime
Пока ты размышляешь, страна уже несколько лет катится ко дну. И прости, может немного накипело, а твоё сообщение довело

**Comment by JeffRockatansky** [comment]
@webknjaz how exactly russians are "planning genocides" on Github?
